matlab/gtsam_examples/Pose2SLAMwSPCG.m

```diff
@@ -54,7 +54,7 @@
 %% Optimize using Levenberg-Marquardt optimization with SubgraphSolver
 params = gtsam.LevenbergMarquardtParams;
 subgraphParams = gtsam.SubgraphSolverParameters;
-params.setLinearSolverType('CONJUGATE_GRADIENT');
+params.setLinearSolverType('ITERATIVE');
```

Comment:
Why?

Reply:
CONJUGATE_GRADIENT has been refactored to ITERATIVE in develop, just no one bothered to check the MATLAB examples.